mmdet/models/detectors/cascade_rcnn.py

```diff
@@ -7,8 +7,9 @@
 from .test_mixins import RPNTestMixin
 from .. import builder
 from ..registry import DETECTORS
-from mmdet.core import (assign_and_sample, bbox2roi, bbox2result, multi_apply,
+from mmdet.core import (bbox2roi, bbox2result,
                         merge_aug_masks)
+from mmdet.core import build_assigner, build_sampler
```

Comment:
Since all methods are imported from the same module, a single statement is enough.